### PR TITLE
Update babylon.js

### PR DESCRIPTION
**Pull request by deltakosh**
Fix extension with Babylon.js when loading gltf1.0 file
(You will need to get a new Babylon.custom.js (because I also fixed an issue with blob loading)

### CODE REVIEW

**Comment by emackey** [comment]
@deltakosh Thanks for this update!  It's an honor to have the creator of BabylonJS here.

We do have a bit of bureaucracy to deal with first.  Do you mind filling out a [CLA (Contributor License Agreement)](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Documentation/Contributors/CLAs/individual-cla-agi-v1.0.txt) (can be plain text file) and email to cla@agi.com?  This just makes it explicit that the code was intended to be shared.  @HowardWolosky sent one already.  Thanks!

**Comment by deltakosh** [comment]
This is my pleasure :)
CLA sent

**Comment by pjcozzi** [comment]
We received @deltakosh's CLA.  Thanks!

**Comment by emackey** [comment]
Thanks again @deltakosh.  This was merged with some changes from @HowardWolosky and published to the marketplace.

**Comment by deltakosh** [comment]
Wonderful!  Thanks team:)

**Comment by emackey** [comment]
@deltakosh Random question if I may.  Would BabylonJS ever support loading "live" glTF contents, as opposed to being given a scene URL?

I ask because I would like this extension to ideally show a live preview, of user-modified but un-saved glTF file contents, in the preview window.  This could mean taking the glTF contents as a string, or an array of objects that came from JSON.parse, etc.  The glTF contents could even be made into a blob and handed off, if needed.

Basically I think it would be helpful to the user to see the effects of their edits before they save them to disk.  But in the serverless Electron environment, I don't have a proper URL or file path to un-saved contents, I just have the raw contents.

**Comment by deltakosh** [comment]
Actually it should already work if you provide blobs and a data string
(See here: https://github.com/BabylonJS/Babylon.js/blob/master/src/Loading/babylon.sceneLoader.ts#L106)

An example for a Babylon file coming from a string: https://www.babylonjs-playground.com/#1AbIIF#7

**Comment by HowardWolosky** [comment]
It turns out that we can't currently leverage the Babylon functionality that David is talking about.  As implemented today, Babylon assumes that any file content that is loaded the way he has suggested is a `.babylon` file, which a `.glTF` file certainly isn't.  I've opened [an Issue](https://github.com/BabylonJS/Babylon.js/issues/2239) against Babylon tracking this problem, along with a proposed solution to it.  I hacked a local copy of Babylon and proved that the rest of the codepath works if Babylon knows that the filecontent is actually `.gltf`. If that issue gets resolved (by me or someone else), then we'll be able to directly load unsaved file content quite easily within this extension.
